### ISSUE TITLE
Added some addresses if interested

### ISSUE THREAD

**Issue by bachalf** [EricJMarti/inventory-hunter#9]
Here are some addresses for the AMD 6000 series if interested:
---
urls:
  - https://www.amazon.com/Sapphire-Radeon-Gaming-Graphics-GDDR6/dp/B08MVCLBWK/
  - https://www.amazon.com/ASUS-Graphics-DisplayPort-Axial-tech-2-9-Slot/dp/B08NWJ29NB/
  - https://www.amazon.com/PowerColor-Graphics-Powered-Raytracing-Infinity/dp/B08MNZM5TK
...

---
urls:
  - https://www.amazon.com/ASUS-Graphics-DisplayPort-Full-Coverage-Radiator/dp/B08NWHKGS9/
  - https://www.amazon.com/Sapphire-Radeon-Gaming-Graphics-GDDR6/dp/B08MVC76SR/
 ...

---
urls:
  - https://www.bestbuy.com/site/xfx-amd-radeon-rx-6800xt-16gb-gddr6-pci-express-4-0-gaming-graphics-card-black/6441226.p?skuId=6441226
  - https://www.bestbuy.com/site/msi-radeon-rx-6800-xt-16g-16gb-gddr6-pci-express-4-0-graphics-card-black-black/6440913.p?skuId=6440913
  - https://www.bestbuy.com/site/xfx-merc-319-amd-radeon-rx-6800xt-16gb-gddr6-pci-express-4-0-gaming-graphics-card-black/6442585.p?skuId=6442585
  
...

---
urls:
  - https://www.bestbuy.com/site/xfx-amd-radeon-rx-6800-16gb-gddr6-pci-express-4-0-gaming-graphics-card-black/6442077.p?skuId=6442077
  - https://www.bestbuy.com/site/msi-radeon-rx-6800-16g-16gb-gddr6-pci-express-4-0-graphics-card-black-black/6441020.p?skuId=6441020
...

---
refresh_interval: 2 # seconds
urls:
  - https://www.newegg.com/asrock-radeon-rx-6800-rx-6800-16g/p/N82E16814930046
  - https://www.newegg.com/asus-radeon-rx-6800-rx6800-16g/p/N82E16814126473
  - https://www.newegg.com/gigabyte-radeon-rx-6800-gv-r68-16gc-b/p/N82E16814932374
  - https://www.newegg.com/msi-radeon-rx-6800-rx-6800-16g/p/N82E16814137608
  - https://www.newegg.com/sapphire-radeon-rx-6800-21305-01-20g/p/N82E16814202389
...

---
refresh_interval: 2 # seconds
urls:
  - https://www.newegg.com/asrock-radeon-rx-6800-xt-rx-6800-xt-16g/p/N82E16814930045
  - https://www.newegg.com/asrock-radeon-rx-6800-xt-rx6800xt-pgd-16go/p/N82E16814930049
  - https://www.newegg.com/asrock-radeon-rx-6800-xt-rx6800xt-tcx-16go/p/N82E16814930050
  - https://www.newegg.com/asus-radeon-rx-6800-xt-rx6800xt-16g/p/N82E16814126472
  - https://www.newegg.com/asus-radeon-rx-6800-xt-rog-strix-lc-rx6800xt-o16g-gaming/p/N82E16814126475
  - https://www.newegg.com/asus-radeon-rx-6800-xt-tuf-rx6800xt-o16g-gaming/p/N82E16814126476
  - https://www.newegg.com/gigabyte-radeon-rx-6800-xt-gv-r68xt-16gc-b/p/N82E16814932373
  - https://www.newegg.com/msi-radeon-rx-6800-xt-rx-6800-xt-16g/p/N82E16814137607
  - https://www.newegg.com/sapphire-radeon-rx-6800-xt-21304-01-20g/p/N82E16814202388
  - https://www.newegg.com/sapphire-radeon-rx-6800-xt-11304-02-20g/p/N82E16814202391
  - https://www.newegg.com/xfx-radeon-rx-6800-xt-rx-68xtacbd9/p/N82E16814150844
  - https://www.newegg.com/powercolor-radeon-rx-6800-xt-axrx-6800xt-16gbd6-2dhce-oc/p/N82E16814131767
  -    
...

There are a couple new ones in newegg that i found but most are just duplicates from you.

**Comment by Wood578Guy** [comment]
Are Amazon and BestBuy supported now?

**Comment by bachalf** [comment]
Sorry, i cannot answer that.  I was just trying to find an AMD card and found these new ones out there and added them to my own .yaml files.
I am also a newbie at this so my experience level is rather low.
Just thought I would attempt to help out someone that has taken the time and trouble to put something like this together.  AMD's launch has been by far worse than Nvidia's 3000 series, which is so hard to believe.

Thank you Eric for all the work you've done and for sharing it with the community.

**Comment by Wood578Guy** [comment]
Fair enough - went and looked and the python does not have support for either...yet.

ANs I echo @bachalf 's gratitude...thank you Eric for making this.

**Comment by rehatiel** [comment]
> And I echo @bachalf 's gratitude...thank you Eric for making this.

I second that.  Anything to give us an edge against these bot bastages.

**Comment by bachalf** [comment]
I found that Best Buy has their own API for searching their stores or online.  You all probably already knew this, but thought i would throw this out there.  I guess others have tried using python scrapers against Best Buy with no avail.  Here is the link to Best Buys API.  You have to sign up to get a key first before you can start using the the API.
https://bestbuyapis.github.io/api-documentation/#overview

**Comment by adamseab** [comment]
Hi folks,

Rather than sticking an issue I thought I'd tag a question on to here... 

As a UK-based Python and Docker n00b (currently on some AWS courses and we've been getting to grips with these tools, so it's actually quite fun to get stuck into something like this), naturally I'm going to be using different, UK-based websites.

I've set up my config yaml files for each of the sites. My query is, as I'm not using BestBuy, Microcentre, BHPhoto etc, do I need to replace all of these classes in the scraper.py file and create new ones for the sites I'm using, or could I just delete them and run with the generic one and see how I get on?

Thanks for everyone's time reading, really good way to get my toes into this sort of thing (and hopefully, cop a 1660 Super for my daughter's PC at the same time, lol!)

**Comment by EricJMarti** [comment]
Thank you all for the kind words!

I am currently working on adding support for Best Buy and Amazon. Best Buy is a bit different from other retailers in that they can detect that a bot is trying to access their website. Fortunately, I have a solution in mind for how to get around it. I have yet to test Amazon, but I suspect they have a similar protection in place.

@bachalf Thanks for these links. I will add them to the repo.

@adamseab Can you tell me which retailers you are using in the UK?

The specific scraper classes such as `NeweggScrapeResult` are intended to detect pricing and look in a specific place for the "add to cart" button. There's also a generic class called `GenericScrapeResult` that searches the entire web page for the "add to cart" text. The generic class is used for websites that don't already have a specific class defined. It works most of the time, but it can break in some scenarios (e.g. when other products are promoted on the same webpage).

**Comment by adamseab** [comment]
@EricJMarti Thank you for the reply! This is such a great project to dabble with :)

I've created config files for...

AWD-IT, Box, CCL, Currys, Overclockers, Scan and TechNextDay - I haven't actually got it running yet. I've been trying to get this running on a headless Raspberry Pi Zero (just so I can leave it ticking over somewhere in the house), but I think the Zero doesn't like Docker, so I'm going to re-try with a Pi 4B, or just let it run on the PC when I'm working :)

Thanks again!

**Comment by kalojero** [comment]
hi @EricJMarti thx for this great tool! 
I'm a spanish user and I want to use this with a spanish website .I've read that there is the GenericScrapeResult and I tried modifying that with the word 'Comprar' that I'm sure that is in the webpage, but the scraper says not in stock. I don't know if maybe I'm running the container with bad configuration or maybe I 've to say that I want to use the genericscraper in some way??, can you put  some light into this plis?? thx a lot

**Comment by bachalf** [comment]
@EricJMarti 
With the 6900 XT coming out on the 8th of Dec, is there a way to get the links to the carss that will be available ahead of the release that way the scrapers can be ready and running prior to the cards being available giving us honest peeps a better chance?

**Comment by EricJMarti** [comment]
Just added Best Buy and Amazon support. Please `git pull` the latest code, rebuilt the Docker image, and restart your running Docker containers.

**Comment by johnnyboy007** [comment]
> @EricJMarti Thank you for the reply! This is such a great project to dabble with :)
> 
> I've created config files for...
> 
> AWD-IT, Box, CCL, Currys, Overclockers, Scan and TechNextDay - I haven't actually got it running yet. I've been trying to get this running on a headless Raspberry Pi Zero (just so I can leave it ticking over somewhere in the house), but I think the Zero doesn't like Docker, so I'm going to re-try with a Pi 4B, or just let it run on the PC when I'm working :)
> 
> Thanks again!

Any chance you can share configs/code changes for Uk sites please ?
